src/js/actions/draganddrop.js

```diff
@@ -52,9 +52,21 @@ define(function (require, exports) {
     registerDroppable.writes = [];
 
     /**
+    * Add many droppables at once
+    *
+    * @param {Immutable.Iterable.List<Object>} list - List of droppable registration information
```

Comment:
Ditto. Also, kill all the dashes. Just start 'splainin' after the variable name.